### ISSUE TITLE
PokeBuddy HUD not fully loading

### ISSUE THREAD

**Issue by OsOCooL** [finndev/PokeBuddy#713]
Everything works good. Its just not loading my level on the top right. Nothing shows on my items, evolve, pokemon and event tabs. It's just blank. 

I tried deleting everything and re extracting the file. Still same results.

**Comment by Thorben24** [comment]
Did you enter your correct login details? If yes what is happening after you press the start button?

**Comment by OsOCooL** [comment]
Everything is running fine. Im still farming, getting poke stops and catching pokemon. It's just the HUD not loading fully

**Comment by Thorben24** [comment]
Well thats strange, never heard that issue before. Could you upload a screenshot?

**Comment by OsOCooL** [comment]
![untitled](https://cloud.githubusercontent.com/assets/20901750/17549703/4256ef62-5e8d-11e6-93d7-7df39ce6a7da.png)

**Comment by OsOCooL** [comment]
^just got IP banned. But even before that the side tabs weren't loading

**Comment by Thorben24** [comment]
Are you sure it was the same then before your ip ban? To be honest i have never seen that before, sorry. The only thing i would know is delete PokeBuddy and download it again.

**Comment by OsOCooL** [comment]
yeah i'll try delete it. Everything was working perfectly. I tried opening another Pokebuddy with another account to see if it was just the account causing the problem. But it wasn't. Then i got IP banned lol

**Comment by OsOCooL** [comment]
Redownloading the file fixed the issue. Thank You

**Comment by MPSv3** [comment]
same problem here... i redownloaded the files but still no working HUD :(

[https://i.gyazo.com/60418e035bccc54ce7df8dae6a604272.png](url)

**Comment by Thorben24** [comment]
@MPSv3 
Also you can do this 2 steps:
Restart your router/modem
Restart your computer.

Often these simple things solve a lot of problems.

**Comment by MPSv3** [comment]
I did both still same issue

**Comment by MPSv3** [comment]
Im on Windows10... somehow i fixed it with starting in Win7 compatibility mode.

**Comment by Thorben24** [comment]
Hum okay. Great to know this.
